pages/_document.js

```diff
@@ -23,6 +23,16 @@ const ANALYTICS_CODE = `
   })();
 `
 
+const GlobalStyle = createGlobalStyle`
+  @font-face {
+    font-family: 'Manrope';
+    src: url('../fonts/Manrope-Regular.ttf');
+    src: url('../fonts/Manrope-Medium.ttf');
+    src: url('../fonts/Manrope-Bold.ttf');
+    src: url('../fonts/Manrope-Light.ttf');
```

Comment:
We should declare these separately (with a `font-weight` for each), otherwise only the last file will get used :-)

Reply:
Also I think there is a variable version of Manrope that we could use? That way we would only need one file fore everything.